sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -34,6 +35,7 @@
  * {@codesnippet com.azure.security.keyvault.keys.keyclient.instantiation}
  *
  * @see KeyClientBuilder
+ * @see PagedIterable
```

Comment:
There is no need to have PagedIterable referenced from the class-level JavaDoc like this. @g2vinay Can you please remove this?